modules/boot/src/utils/hook.cpp

```diff
@@ -197,14 +200,27 @@ int saveInjectHook(void* i_scene) {
 // Hook to disable save inject flags after dScnPly phase_4 is run
 int endSaveInjectHook(void* i_scene) {
     int rt = dScnPly__phase_4Trampoline(i_scene);
+    s16 scene = fpcM_GetName(i_scene);
 
+    if (scene == PROC_OPENING_SCENE ) {
+        g_PreLoopListener->addListener(GZ_endlessNightOnTitle);
+    }
```

Comment:
idk if this is a very good place for this. it'd probably be better hooked onto something like dComIfG_resetToOpening or dComIfG_changeOpeningScene.

Reply:
i don't see a better way to handle this because if a save is loaded in title screen i don't see how remove the listner